src/Fixer/ControlStructure/TrailingCommaInMultilineFixer.php

```diff
@@ -84,27 +85,34 @@ public function getDefinition(): FixerDefinitionInterface
 
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isAnyTokenKindsFound([T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN, '(']);
+        return $tokens->isAnyTokenKindsFound([T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN, '(', CT::T_DESTRUCTURING_SQUARE_BRACE_OPEN]);
     }
 
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
+        $allOptions = [
+            self::ARRAY_DESTRUCTURING,
+            self::ELEMENTS_ARGUMENTS,
+            self::ELEMENTS_ARRAYS,
+            self::ELEMENTS_PARAMETERS,
+            self::MATCH_EXPRESSIONS,
+        ];
+
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('after_heredoc', 'Whether a trailing comma should also be placed after heredoc end.'))
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
-            (new FixerOptionBuilder('elements', sprintf('Where to fix multiline trailing comma (PHP >= 8.0 for `%s` and `%s`).', self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS))) // @TODO: remove text when PHP 8.0+ is required
+            (new FixerOptionBuilder('elements', sprintf('Where to fix multiline trailing comma (PHP >= 8.0 for `%s` and `%s`).', self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS))) // @TODO: update text when PHP 8.0+ is required
                 ->setAllowedTypes(['array'])
-                ->setAllowedValues([new AllowedValueSubset([self::ELEMENTS_ARRAYS, self::ELEMENTS_ARGUMENTS, self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS])])
+                ->setAllowedValues([new AllowedValueSubset($allOptions)])
                 ->setDefault([self::ELEMENTS_ARRAYS])
                 ->setNormalizer(static function (Options $options, $value) {
                     if (\PHP_VERSION_ID < 8_00_00) { // @TODO: drop condition when PHP 8.0+ is required
-                        foreach ([self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS] as $option) {
-                            if (\in_array($option, $value, true)) {
-                                throw new InvalidOptionsForEnvException(sprintf('"%s" option can only be enabled with PHP 8.0+.', $option));
-                            }
-                        }
+                        $value = array_filter(
```

Comment:
why? for 7.4 , we were throwing exception,
now - only ignoring the value and moving on

Reply:
because I added it to the set, meaning if you want to follow the set but you are in 7.4 it would throw the exception

Reply:
Also, there is no much of a point how it done now from user POV to throw the exception.
ff you are on PHP7.4 and configure it throws an exception.
If we wouldn't throw, than if there is an enum in the passed code the linter doesn't pass and already throws an exception.
If there isn't an enum and we would ignore the config setting, the fixer works just fine.

We could also check/ignore the option in the set, but that makes things even more complex.